File: 1748-sum-of-unique-elements/1748-sum-of-unique-elements.java
class Solution {
    public int sumOfUnique(int[] nums) {
       int cnt[]=new int[101]; 
        int res = 0;
    for (int n : nums)
        ++cnt[n];
    for (int i = 1; i <= 100; ++i)
        if (cnt[i] == 1)
            res += i;
    return res;  
    }
}

File: 1748-sum-of-unique-elements/1748-sum-of-unique-elements.cpp
class Solution {
public:
    int sumOfUnique(vector<int>& nums) {
        unordered_map<int,int> m;
        for(auto i:nums)
            m[i]++;
        int sum=0;
        for(int i=0;i<nums.size();++i)
        {
            if(m[nums[i]]<=1)
                sum+=nums[i];
        }
        return sum;
    }
};